src/Analyzers/CSharp/CodeFixes/UseNullPropagation/CSharpUseNullPropagationCodeFixProvider.cs

```diff
@@ -32,6 +33,32 @@ public CSharpUseNullPropagationCodeFixProvider()
         {
         }
 
+        protected override bool TryGetBlock(SyntaxNode? statement, [NotNullWhen(true)] out StatementSyntax? block)
+        {
+            if (statement is BlockSyntax stetementBlock)
+            {
+                block = stetementBlock;
+                return true;
+            }
+
+            block = null;
+            return false;
+        }
+
+        protected override StatementSyntax ReplaceBlockStatements(StatementSyntax block, StatementSyntax newInnerStatement)
+        {
+            var newStatementList = SyntaxFactory.SingletonList(newInnerStatement);
+            return ((BlockSyntax)block).WithStatements(newStatementList);
+        }
+
+        protected override SyntaxNode PostProcessElseIf(IfStatementSyntax ifStatement, StatementSyntax newWhenTrueStatement)
+        {
+            var elseClauseSyntax = (ElseClauseSyntax)ifStatement.Parent!;
+            return elseClauseSyntax
+                    .WithElseKeyword(elseClauseSyntax.ElseKeyword.WithTrailingTrivia())
```

Comment:
Dedent

Reply:
You just meant "decent", right?

Reply:
No.  Dedent one level. Indent should be 4 not 8. Thanks.

Reply:
I didn't know such word even exists...